Blocked on:
- #107519

https://github.com/zephyrproject-rtos/zephyr/actions/runs/24588250420/job/71903123213?pr=107507
```
=== GitHub Access Check ===
Collaborators without at least triage access:
  - sylvioales
Some added maintainers or collaborators do not have sufficient access.
Error: sylvioales does not have needed access level to zephyrproject-rtos/zephyr
Error: Process completed with exit code 1.
```

--- a/MAINTAINERS.yml
+++ b/MAINTAINERS.yml
@@ -5829,6 +5829,7 @@ West:
     - mbolivar
     - carlescufi
     - marc-hb
+    - sylvioalves
   files:
     - scripts/west-commands.yml
     - scripts/west_commands/

